README.md

```diff
@@ -49,6 +49,11 @@ In case you need to use a different host, you can also set it up as an environme
 export SF_HOST=...
 ```
 
+## Important considerations
+
+- Template handles should be unique, since we based on them to identify the correct template when there is still no reference to the corresponding ID. The only exception is templates that are included from a marketplace template (so you could have in your environment one template "local" and one from the marketplace using the same handle, the CLI will skip the one from the marketplace and try to indentify the local one).
```

Comment:
```suggestion
- Reconciliation handles should be unique, since we identify the correct reconciliation based on these handles when there is no reference to the corresponding ID. The only case where duplicate handles are supported in a single firm, are reconciliations that are added from a marketplace reconciliation (so you could have one custom reconciliation and one from the marketplace using the same handle, the CLI will skip the one from the marketplace and try to identify the custom one).
```